data/phones/HOWTO.md

```diff
@@ -16,12 +16,14 @@ file:
 -   non-native segments (e.g., a transcription of *Bach* as ending in the
     voiceless velar fricative /x/)
 
-Finally, for phonemic lists, there may be segments that are properly considered
-pure allophones but appear in phonemic transcriptions. However, such segments
-may be quite frequent in the data and removing all pronunciations that contain
-them would greatly reduce the amount of available data. Therefore, we prefer to
-simply add a comment of the form `# Allophone of ...`; such annotations will
-ultimately be used improve Wiktionary itself.
+When creating a `.phones` file for broadly transcribed data, the goal is often
+to create something that approximates a list of phonemes. However, there may 
+be segments that are properly considered pure allophones but appear in broad 
+transcriptions. However, such  segments may be quite frequent in the data and 
```

Comment:
I like what you wrote here but very small nit-pick: there are two spaces after the word "such"